packages/rum-core/src/common/service-factory.js

```diff
@@ -26,42 +26,43 @@
 import ApmServer from './apm-server'
 import ConfigService from './config-service'
 import LoggingService from './logging-service'
-import { CONFIG_CHANGE } from './constants'
+import {
+  CONFIG_CHANGE,
+  CONFIG_SERVICE,
+  LOGGING_SERVICE,
+  APM_SERVER
+} from './constants'
+import { __DEV__ } from '../env'
+
+const serviceCreators = {
```

Comment:
This is exactly similar to file level instance. Remember you were suggesting aganist this idea? This would mean we have the same factory instance for all the tests? Previously `createServiceFactory` would give us a fresh instance every time.

Reply:
Although, this is a file level object, it's not the same as I was suggesting.

The file level instance that I don't like is the one that contains our service instances, imagine if `ServiceFactory.instances` was a file level object, this is not a good pattern.

But serviceCreators are only functions that create the services, the only way this file level object can make problems is as if we start changing it during our tests.